docs/upgrade/2.6.1_to_2.7.0.rst

```diff
@@ -0,0 +1,83 @@
+.. _latest_upgrade_guide:
+
+Upgrade from 2.6.1 to 2.7.0
+===========================
+
+Update Servers to SecureDrop 2.7.0
```

Comment:
Should we copy/reference the SHA-1 key stuff from https://securedrop.org/news/securedrop-2_7_0-pre-release-announcement/ here?